kbase-extension/static/kbase/js/widgets/narrative_core/upload/fileUploadWidget.js

```diff
@@ -30,6 +32,13 @@ define([
             this.path = options.path;
             this.stagingUrl = Config.url('staging_api_url');
             this.userInfo = options.userInfo;
+
+            var runtime = Runtime.make();
+            this.stagingServiceClient = new StagingServiceClient({
```

Comment:
Good idea to put it into the init. Maybe we need to refactor the client in uploadGlobusClickEvent to reuse this client as well.

Reply:
Good call. I just did that :)